packages/teleport/src/Apps/AddApp/Automatically/Automatically.tsx

```diff
@@ -180,20 +189,18 @@ const requiredAppName = value => () => {
     };
   }
 
-  try {
-    const tmp = new URL(`https://${value}`);
-    if (tmp.hostname !== value) {
-      throw new Error();
-    }
+  if (value.length > DNS1035LABEL_MAXLENGTH) {
+    return {
+      valid: false,
+      message: 'Must be 63 chars or less',
+    };
+  }
 
-    // cannot be a sub-domain
-    if (tmp.hostname.split('.').length > 1) {
-      throw new Error();
-    }
-  } catch {
+  const match = value.match(REGEX_DNS1035LABEL);
+  if (!match) {
     return {
       valid: false,
-      message: 'Invalid',
+      message: 'Invalid name',
```

Comment:
lets change it to `Should be a valid DNS sub-domain name`

Reply:
could you also ensure that the text inside app name field is also a lower case? 

I think the css rule would do.

```
text-transform: lowercase;
```

Reply:
revised as discussed in slack